marketstack/src/endpoint/eod.ts

```diff
@@ -0,0 +1,42 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { AdapterRequest, Config } from '@chainlink/types'
+import { DEFAULT_INTERVAL, DEFAULT_LIMIT } from '../config'
+import { util } from '@chainlink/ea-bootstrap'
+
+export const Name = 'eod'
+
+const customError = (data: any) => data.Response === 'Error'
+
+const customParams = {
+  base: ['base', 'from', 'coin'],
+  interval: false,
+  limit: false,
+}
+
+export const execute = async (config: Config, request: AdapterRequest) => {
+  const validator = new Validator(request, customParams)
+  if (validator.error) throw validator.error
+
+  const symbols = validator.validated.data.base.toUpperCase()
+  const interval = validator.validated.data.interval || DEFAULT_INTERVAL
+  const limit = validator.validated.data.limit || DEFAULT_LIMIT
+  const url = `eod`
+
+  const params = {
+    symbols,
+    interval,
+    limit,
+    access_key: util.getRandomRequiredEnv('API_KEY'),
+  }
+
+  const reqConfig = {
+    ...config.api,
+    params,
+    baseURL: 'http://api.marketstack.com/v1/',
```

Comment:
I would expect the `baseURL` and `API_KEY` to be the same no matter what endpoint is being hit. These might be better injected in Config.ts,